lib/survey_execution/initializer.rb

```diff
@@ -0,0 +1,12 @@
+module SurveyExecution
+  class Initializer
+    def initialize(survey, respondents)
+      @client = Sms::Client.new
```

Comment:
Not too familiar with Twilio client but do we really want to instantiate a new one per initializer? As opposed to having one client object for our entire application?